notebooks/NYC_HEALTH_TESTS.ipynb

```diff
@@ -0,0 +1,197 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# NYCHealth Coronavirus (COVID-19) data\n",
+    "### Original datasource: https://github.com/nychealth/coronavirus-data"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import pandas as pd\n",
+    "import numpy as np\n",
+    "import requests\n",
+    "import json\n",
+    "import re\n",
+    "import csv\n",
+    "from airflow.models import Variable\n",
```

Comment:
notebooks should be able to run without airflow dependencies. Please use OS env variables